src/Request/JsonConvertibleParamConverter.php

```diff
@@ -93,23 +90,27 @@ public function supports(ParamConverter $configuration)
     }
 
     /**
-     * @param string $camelCase
-     * @return string
-     * @see \Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter
+     * Deeply converts an object's property names to camel case. Doesn't support circular references!
+     *
+     * @param object $object
+     * @return object
      */
-    private function camelCaseToSnakeCase($camelCase)
+    private function camelCaseObjectPropertyNames($object)
     {
-        $snakeCase = '';
-
-        $len = strlen($camelCase);
-        for ($i = 0; $i < $len; $i++) {
-            if (ctype_upper($camelCase[$i])) {
-                $snakeCase .= '_'.strtolower($camelCase[$i]);
-            } else {
-                $snakeCase .= strtolower($camelCase[$i]);
+        $camelCasedObject = (object) [];
```

Comment:
just create a `\stdClass` then, as this leads to the same but in a less obvious way.